app/sequencescape/sequencescape/api/v2.rb

```diff
@@ -11,11 +11,11 @@ module Sequencescape::Api::V2
     { child_plates: :purpose },
     {
       wells: [
+        :qc_results,
         {
           downstream_tubes: 'purpose',
           requests_as_source: %w[request_type primer_panel pre_capture_pool submission],
           aliquots: ['sample.sample_metadata', { request: %w[request_type primer_panel pre_capture_pool submission] }],
-          qc_results: [],
```

Comment:
Just trying to understand - qc_results is supposed to be a default include but because its including no fields `[]` nothing is returned?

Reply:
Ok cool sounds good to me. Think this probably solves: https://github.com/sanger/limber/issues/2292 then. Ill pull it down and give it a quick test